src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -20,23 +22,41 @@ import RRWebIntegration from 'app/components/events/rrwebIntegration';
 import EventSdkUpdates from 'app/components/events/sdkUpdates';
 import {DataSection} from 'app/components/events/styles';
 import EventUserFeedback from 'app/components/events/userFeedback';
-import {t} from 'app/locale';
+import ExternalLink from 'app/components/links/externalLink';
+import {t, tct} from 'app/locale';
 import space from 'app/styles/space';
-import {Group, Organization, Project, SharedViewOrganization} from 'app/types';
-import {Entry, Event} from 'app/types/event';
+import {
+  ExceptionValue,
+  Group,
+  Organization,
+  Project,
+  SharedViewOrganization,
+} from 'app/types';
+import {DebugFile} from 'app/types/debugFiles';
+import {Image} from 'app/types/debugImage';
+import {Entry, EntryType, Event} from 'app/types/event';
+import {Thread} from 'app/types/events';
 import {isNotSharedOrganization} from 'app/types/utils';
-import {objectIsEmpty} from 'app/utils';
+import {defined, objectIsEmpty} from 'app/utils';
 import {analytics} from 'app/utils/analytics';
+import withApi from 'app/utils/withApi';
 import withOrganization from 'app/utils/withOrganization';
+import {projectProcessingIssuesMessages} from 'app/views/settings/project/projectProcessingIssues';
 
+import findBestThread from './interfaces/threads/threadSelector/findBestThread';
+import getThreadException from './interfaces/threads/threadSelector/getThreadException';
 import EventEntry from './eventEntry';
 
+const MATCH_MINIFIED_DATA_REGEX = /^(\w|\w{2}\.\w{1,2}|\w{3}((\.\w)|(\.\w{2}){2}))(\.|$)/g;
```

Comment:
Should we rename this variable or add a comment to be more clear that this is built for minified java stack trace detection?